frontend/src/components/organisms/TableFilters/index.jsx

```diff
@@ -41,10 +64,22 @@ const TableFilter = (props) => {
     updateSelectedFilter,
   } = props;
 
-  const category =
-    filterData?.length > 0 &&
-    categoryFilterValue &&
-    filterData.filter((item) => item.name === categoryFilterValue)?.[0]?.key;
+
+  tableFilters.forEach(function(entry) {
+    console.log(entry);
+  });
+
+
+  const getFilterID = (name) => {
+    if (!(tableFilters?.length > 0)) {
+      return "no-filters";
+    }
+    const filterResults = tableFilters.filter((item) => item.name === name);
```

Comment:
Without reading all the action and reducer code ... is this really saying that the previous `filterData` is just the same as `tableFilters`??